locale/en/blog/weekly-updates/weekly-update.2016-04-17.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Weekly Update - Apr 17th, 2016
+author: Minwoo Jung (github.com/JungMinu)
+date: 2016-04-17T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-04-17
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v4.4.3 (LTS) is released.
+
+### Node v4.4.3 (LTS) Releases
+
+We have one releases: [Node v4.4.3 (LTS)](https://nodejs.org/en/blog/release/v4.4.3/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### New Node.js Foundation Survey Reports New “Full Stack” In Demand Among Enterprise Developers
+
+The Node.js User Survey report features insights on emerging trends happening in this massive community that serves as a leading indicator on trends like microservices architectures, real-time web applications, Internet of Things (IoT). The report paints a detailed picture of the technologies that are being used, in particular, with Node.js in production and language preferences (current and future) for front end, back end and IoT developers.
+
+See https://nodejs.org/en/blog/announcements/nodejs-foundation-survey/ for more information.
+
+### Community Updates
+
+* [NodeUp podcast episode 100]: A Creative Coding Show with Mariko Kosaka, Kassandra Perch, Myles Borins, and Rich Trott.(http://nodeup.com/onehundred)
```

Comment:
Parentheses should come right after the brackets for Markdown to render correctly here and elsewhere...